orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -5252,7 +5252,7 @@ def run_step(model, x):
     assert ort_model._torch_module._execution_manager(True)._runtime_options.onnx_opset_version == 13
 
 
-@pytest.mark.parametrize("opset_version", [12, 13, 14, 15])
+@pytest.mark.parametrize("opset_version", [12, 13, 14, 15, 20])
```

Comment:
Since 12, 13, 14 are quite old I am wondering whether we can remove them and save some test time. Do you know whether any of our partners use older opsets?